plugins/calibre.koplugin/metadata.lua

```diff
@@ -8,34 +8,42 @@ of storing it.
 @module koplugin.calibre.metadata
 --]]--
 
+local lfs = require("libs/libkoreader-lfs")
 local rapidjson = require("rapidjson")
 local logger = require("logger")
+local parser = require("parser")
 local util = require("util")
 
-local unused_metadata = {
-    "application_id",
-    "author_link_map",
-    "author_sort",
-    "author_sort_map",
-    "book_producer",
-    "comments",
-    "cover",
-    "db_id",
-    "identifiers",
-    "languages",
-    "pubdate",
-    "publication_type",
-    "publisher",
-    "rating",
-    "rights",
-    "thumbnail",
-    "timestamp",
-    "title_sort",
-    "user_categories",
-    "user_metadata",
-    "_series_sort_",
+local used_metadata = {
+    "uuid",
+    "lpath",
+    "last_modified",
+    "size",
+    "title",
+    "authors",
+    "tags",
+    "series",
+    "series_index"
 }
 
+local function slim(book)
+    local slim_book = {}
+    for k, _ in pairs(used_metadata) do
```

Comment:
It's an array, you can go with `ipairs` for that infinitesimal speed bump ;p.

Reply:
indeed, it's an array and that's a typo from refactor. It should be `ipairs` for sure

Reply:
Yeah, noticed the "same" one used later in `parser` is an `ipairs` ;).

Reply:
FWIW, it is infinitesimal speed gain for a single book. For 50000 books managed using rapidjson the bump is noticeable (6 vs 8 seconds on my computer)

Reply:
(Mildly related: definitely like the logic swap from a blacklist to a whitelist for the field! :+1:)